src/proxy.js

```diff
@@ -113,13 +112,18 @@ class Tracer extends BaseTracer {
     return this._tracer.extract.apply(this._tracer, arguments)
   }
 
+  scopeManager () {
+    this._deprecate('scopeManager')
+    return this._tracer.scopeManager.apply(this._tracer, arguments)
```

Comment:
should this be `this._tracer.scope.apply` ?

Reply:
This calls the old class which is now a no-op.